index.js

```diff
@@ -41,22 +44,25 @@ getBooksPublishedAfter2010([
 // Q3
 // Write a JavaScript function that takes an array of product objects and returns the total price of all products in the category "electronics".
 function getTotalPriceOfElectronics(products) {
-  // Write code here
+  const electronics = products.map((product) => product.price);
+  return electronics;
 }
 
 getTotalPriceOfElectronics([
   { name: "Laptop", category: "electronics", price: 999 },
   { name: "Book", category: "books", price: 30 },
   { name: "Smartphone", category: "electronics", price: 699 },
+  console.log(getTotalPriceOfElectronics),
```

Comment:
you can't add the console.log inside of an object, because it will result in a TypeError when passing it to getTotalPriceOfElectronics, undefined.price will result in a TypeError